style.css

```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body {
+  background: rgb(25, 25, 43);
+}
+
+.header {
+  position: absolute;
+  width: 100vw;
+  height: 7vh;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 6px;
+  position: absolute;
+  width: 95px;
+  height: 40px;
+  left: 10px;
+}
+
+.logo {
+  width: 98px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.hamburger {
+  position: absolute;
+  width: 20px;
+  height: 15px;
+  right: 10px;
+  top: 10px;
+}
+
+.line-1 {
+  position: absolute;
+  width: 16px;
+  top: 7px;
+  left: 4px;
+  border: 2px solid white;
+  border-radius: 15px;
+}
+
+.line-2 {
+  position: absolute;
+  width: 16px;
+  top: 12px;
+  left: 4px;
+  border: 2px solid white;
+  border-radius: 15px;
+}
+
+.line-3 {
+  position: absolute;
+  width: 16px;
+  top: 17px;
+  left: 4px;
+  border: 2px solid white;
+  border-radius: 15px;
+}
+
+.landing-page {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+}
+
+.large-text {
+  width: 300px;
+  height: 120px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 55px;
+  line-height: 60px;
+  color: aquamarine;
+}
+
+.medium-text {
+  width: 350px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 25px;
+  line-height: 30px;
+  display: flex;
+  align-items: center;
+  color: goldenrod;
+  flex: none;
+}
+
+.small-text {
+  width: 400px;
+  height: 120px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: goldenrod;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.socials {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 250px;
+  height: 20px;
+}
+
+.twitter {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 20px;
+  height: 20px;
+}
+
+.github {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 20px;
+  height: 20px;
+}
+
+.medium {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 20px;
+  height: 20px;
+}
```

Comment:
- [ ] You didn't follow the Figma design Requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] For Minimum mobile screen size should at 375px. But Your  paragraph("small text" ) is not fully visible and responsive . Kindly fix it.
Your Headline section Look likes : 
![h](https://user-images.githubusercontent.com/93598189/185465547-916a398f-0b68-4e49-a5ee-6049c08f7737.PNG)

Figma Headline section look likes : 

![Fi](https://user-images.githubusercontent.com/93598189/185466122-f560ea44-af8b-4df1-9fd6-a3b50867f22e.PNG)